### PR TITLE
[RSPECT CONTROLLER] Sessions controler rspec

### PR DESCRIPTION
**Pull request by huytn-1183**
## Related Tickets
- [#43123](https://edu-redmine.sun-asterisk.vn/issues/43123)

## WHAT (optional)
- Sessions Contrller RSpec login, logout test

## HOW
- Rspec request

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by huytn-1183** [comment]
ready

**Comment by huytn-1183** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -42,6 +42,8 @@ def remember user
   end
 
   def forget user
+    return if user.blank?
```
Comment (vanvtt-0952) [inline app/helpers/sessions_helper.rb:45]:
```suggestion
    return unless user.present?
```

**Comment by huytn-1183** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**